modules/ossm-cr-pilot.adoc

```diff
@@ -5,18 +5,42 @@
 [id="ossm-cr-pilot_{context}"]
 = Istio Pilot configuration
 
-Here is an example that illustrates the Istio Pilot parameters for the `ServiceMeshControlPlane` and a description of the available parameters with appropriate values.
+You can configure Pilot to schedule or set limits on resource allocation. 
+The following example illustrates the Pilot parameters for the `ServiceMeshControlPlane` and a description of the available parameters with appropriate values.
 
 .Example pilot parameters
 [source,yaml]
 ----
-  pilot:
-    resources:
-      requests:
-        cpu: 100m
-        memory: 128Mi
-    autoscaleEnabled: false
-    traceSampling: 100
+spec:
+  runtime:
+    components:
+      tracing.jaeger.elasticsearch: {}
```

Comment:
We should probably get rid of tracing.jaeger.elasticsearch here, as we're explicitly talking about pilot runtime config.